Remotion/Web/Development.Analyzers.IntegrationTests/RMWEB0001_WebStringAnalyzerTests.cs

```diff
@@ -0,0 +1,246 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Verify = Remotion.Web.Development.Analyzers.Tests.CSharpAnalyzerVerifier<Web.Development.Analyzers.RMWEB0001_WebStringAnalyzer>;
```

Comment:
`Verifier` not `Verify` (and I'm happy if there's a better naming option). To me, `Verify.VerifyAnalyzerAsync` looks strange and it doesn't for the regular naming convention of using nouns for types. I think only fluent APIs and Assert get this exemption from the rule.